### PR TITLE
TMS Service git-history fixes

### PR DESCRIPTION
**Pull request by joeygoksu**

### CODE REVIEW

File: tms-service/src/app.ts [hunk 1]
```diff
@@ -21,16 +22,16 @@ class App extends Koa {
   }
 
   configureMiddlewares(): void {
-    const readSwagger = swagger.loadDocumentSync('./mojaloop-api.yaml');
+    const readSwagger = swagger.loadDocumentSync(path.join(__dirname, 'mojaloop-api.yaml'));
```
Comment (johanfol) [inline tms-service/src/app.ts:25]: Why was this change needed?

File: tms-service/src/controllers/execute.ts [hunk 2]
```diff
@@ -23,8 +15,6 @@ export const monitorTransaction = async (ctx: Context): Promise<Context> => {
 
     ctx.body = { result: resp.getBody() };
   } catch (e) {
-    logger.error(e);
```
Comment (johanfol) [inline tms-service/src/controllers/execute.ts:26]: Why remove logging of the error here?

File: tms-service/src/controllers/healthcheck.ts [hunk 3]
```diff
@@ -41,8 +41,6 @@ export const healthCheck = async (ctx: Context, next: Next): Promise<void | Cont
 
     await next();
   } catch (error) {
-    logger.error(error);
-
-    ctx.body = { result: 'Something went wrong' };
+    ctx.body = { result: 'Something went wrong', error };
```
Comment (johanfol) [inline tms-service/src/controllers/healthcheck.ts:44]: Why remove logging of the error here?

**Review by johanfol — CHANGES_REQUESTED** [review]

File: tms-service/src/app.ts (hunk 1 above)
Comment (joeygoksu) [reply tms-service/src/app.ts:25]: It was causing the import error after the build. Due to that, I converted to absolute import rather than a relative one.

**Review by johanfol — APPROVED** [review]
